README.md

```diff
@@ -3,14 +3,16 @@ Demonstrates what happens when you send a QUBO to the D-Wave QPU
 
 ## Overview
 
-This program takes a 3-variable QUBO function and demonstrates what goes on "behind the scenes" as it runs on the QPU.  We see:
+This program takes a 3-variable QUBO function and demonstrates what goes on 
+"behind the scenes" as it runs on the QPU.  We see:
 
 1. The original QUBO.
 2. The equivalent Ising form.
-3. An embedded Ising form onto the QPU architecture using four qubits.
+3. An Ising form embedded onto the QPU architecture using four qubits.
```

Comment:
I think this should read "The Ising form..." rather than "An Ising form..." since we are using the specific one from step 2, yes?

Reply:
This is fixed - I pushed it. Thanks.

Reply:
@vgoliber, I cropped the other photo. In terms of the one with the line break, alas, it was formulated on a half slide, and I am having trouble with the PowerPoint. I didn't solve it yet. Let me know how critical it is.